docs/app/docs/overlay/sonner/page.mdx

```diff
@@ -0,0 +1,132 @@
+import { Callout, Tabs } from 'nextra/components';
+import { Widget } from '@/components/demo/widget.tsx';
+import LinkBadge from '@/components/ui/link-badge/link-badge.tsx';
+import LinkBadgeGroup from '@/components/ui/link-badge/link-badge-group.tsx';
+
+# Sonner
+
+An opinionated toast that temporarily displays a succinct message. 
+
+To show a toast, use the `showFToast(...)` function. The function must be called from a widget that is a descendant of a
+`FSonner` or `FScaffold` widget.
+
+<LinkBadgeGroup>
+    <LinkBadge label="API Reference" href="https://pub.dev/documentation/forui/latest/forui.widgets.sonner/"/>
+</LinkBadgeGroup>
+
+<Tabs items={['Preview', 'Code']}>
+  <Tabs.Tab>
+    <Widget name='sonner' query={{}} height={620}/>
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```dart {13, 37, 60} copy
+    class SonnerPage extends StatelessWidget {
+      @override
+      Widget build(BuildContext context) {
+        final cardStyle = context.theme.cardStyle.copyWith(
+          contentStyle: context.theme.cardStyle.contentStyle.copyWith(
+            titleTextStyle: context.theme.typography.sm.copyWith(
+              color: context.theme.colors.primary,
+              fontWeight: FontWeight.w600,
+            ),
+          ),
+        );
+
+        return FSonner(
+          child: Builder(
+            builder: (context) => Expanded(
+              child:
+               Center(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  spacing: 5,
+                  children: [
+                    FButton(
+                      intrinsicWidth: true,
+                      onPress: () {
+                        Widget buildToast(BuildContext context, FToast toast) => IntrinsicHeight(
+                          child: FCard(
+                            style: cardStyle,
+                            title: const Text('Event has been created'),
+                            subtitle: const Padding(
+                              padding: EdgeInsets.symmetric(vertical: 5),
+                              child: Text('This is a brief description that provides additional context.'),
+                            ),
+                            child: FButton(onPress: () => toast.dismiss(), child: const Text('Undo')),
+                          ),
+                        );
```

Comment:
Nit, but could we make the design more similar to what they have on shadcn? I think it generally looks a lot more polished and how a toast should look like:
https://ui.shadcn.com/docs/components/sonner

In addition, it could be more meaningful to let the toast appear at different corners of the screen similar to sheet:
https://forui.dev/docs/overlay/sheet